healthcheck/sqlhealthcheck.go

```diff
@@ -106,10 +106,17 @@ func PreformHealthChecks(healthChecks Format, cxn *sql.DB) (results []SQLHealthC
 
 // RunHealthCheck runs through a single healthcheck and saves the result
 func RunHealthCheck(healthCheck SQLHealthCheck, cxn *sql.DB) SQLHealthCheck {
-	rows, _ := cxn.Query(healthCheck.Query)
-	var answer string
+	answer := ""
+
+	rows, err := cxn.Query(healthCheck.Query)
+
+	if err != nil {
+		log.Fatalf("Error: %v", err)
+	}
+
```

Comment:
Yep, I did this as well to ensure errors showed up loudly, we should definitely handle this more holistically.